message RpcError {
    required string message = 1;
}